Mathlib/Data/Complex/Basic.lean

```diff
@@ -76,10 +76,10 @@ theorem range_im : range im = univ :=
 /-- The natural inclusion of the real numbers into the complex numbers.
 The name `Complex.ofReal` is reserved for the bundled homomorphism. -/
```

Comment:
Please update that one also :-)

Reply:
I simply deleted that sentence. It was there to justify the bad naming convention, which is now gone ✨